README.md

```diff
@@ -324,6 +324,7 @@ Table of Contents
   * [azure.microsoft.com](https://azure.microsoft.com/) — MS Azure gives USD 200 worth of free usage for a trial
   * [appharbor.com](https://appharbor.com/) — A .Net PaaS that provides 1 free worker
   * [heroku.com](https://www.heroku.com/) — Host your apps in the cloud, free for single process apps
+  * [hasura.io](https://www.hasura.io/) — Platform to build and deploy app backends fast, free on own infrastructure
```

Comment:
this looks really nice, unfortunately though this list is for SaaS - ie. not self hosted.  Once they launch their SaaS and there is a free tier then for sure it would be a good fit

thanks for the effort though!